clang/lib/Driver/Driver.cpp

```diff
@@ -4289,8 +4289,8 @@ void Driver::BuildActions(Compilation &C, DerivedArgList &Args,
       // Unbundler only handles objects.
       if (auto *IA = dyn_cast<InputAction>(LI)) {
         std::string FileName = IA->getInputArg().getAsString(Args);
-        if (IA->getType() == types::TY_Object &&
-            !isObjectFile(FileName))
+        if ((IA->getType() == types::TY_Object && !isObjectFile(FileName)) ||
```

Comment:
Minor/disregardable:
```suggestion
        if (IA->getType() == types::TY_Object && !isObjectFile(FileName) ||
```

Reply:
100% OK if it's to avoid warnings, please disregard